h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -47,9 +47,30 @@
 from h2o.utils.typechecks import is_type
 import datetime
 import time # needed to randomly generate time
+import threading
 import uuid # call uuid.uuid4() to generate unique uuid numbers
 
 
+class Timeout:
+
+    def __init__(self, timeout_secs, on_timeout=None):
+        enabled = timeout_secs is not None and timeout_secs >= 0
+        self.timer = threading.Timer(timeout_secs, on_timeout) if enabled else None
+
+    @property
+    def active(self):
```

Comment:
@seb-h2o is this method being used?

Reply:
no, you're right, I should remove this: I took this from my utilities in the benchmark app :)